tsdef/RemoteAudioTrack.d.ts

```diff
@@ -0,0 +1,6 @@
+import { AudioTrack } from './AudioTrack';
+import { Track } from './Track';
+
+export class RemoteAudioTrack extends AudioTrack {
+  sid: Track.SID;
```

Comment:
looks like its missing some members like:  `isSwitchedoff` `priority`
For any public types please confirm that they match what we see on our docs: 'https://media.twiliocdn.com/sdk/js/video/releases/2.7.2/docs/RemoteAudioTrack.html